src/server/system_services/schemas/bucket_schema.js

```diff
@@ -269,6 +269,22 @@ module.exports = {
                 reads: { type: 'integer' },
                 writes: { type: 'integer' },
             }
-        }
+        },
+        // Do we need enable/disable option? if yes, how to do that?
```

Comment:
according to [AWS docs](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketLogging.html) you disable logging by sending an empty logging configuration, so I think it's equal to just deleting an existing one. so in my opinion there is no need to have enable\disable in this case.

in general when we do need it we can just add a boolean flag `enabled` or something similar

Reply:
you can remove the comment

Reply:
@aspandey, remove the comment before merging